draft-ietf-quic-transport.md

```diff
@@ -3872,7 +3872,9 @@ Destination Connection ID of subsequent packets that it sends.
 
 A server MAY send Retry packets in response to Initial and 0-RTT packets.  A
 server can either discard or buffer 0-RTT packets that it receives.  A server
-can send multiple Retry packets as it receives Initial or 0-RTT packets.
+can send multiple Retry packets as it receives Initial or 0-RTT packets.  A
+server MUST NOT send more than one Retry packet in response to a single UDP
+datagram.
```

Comment:
And this one.

Reply:
In what use case are you expecting to send multiple Retrys in response to a UDP datagram?  And in those cases, can we require coalescing the Retrys into a single datagram, and so we could say no more than one UDP datagram in response?